include/tblite/calculator.h

```diff
@@ -113,6 +113,12 @@ tblite_get_calculator_shell_map(tblite_context /* ctx */,
                             tblite_calculator /* calc */,
                             int* /* sh2at */);
 
+/// Query calculator for angular momenta of shells
+TBLITE_API_ENTRY void TBLITE_API_CALL
+tblite_get_calculator_angular_momenta(tblite_context /* ctx */,
+                              tblite_calculator /* calc */,
+                              int* /* am */);
```

Comment:
Hanging indentation for C:

```suggestion
tblite_get_calculator_angular_momenta(tblite_context /* ctx */,
                                      tblite_calculator /* calc */,
                                      int* /* am */);
```